manager/allocator/networkallocator/networkallocator.go

```diff
@@ -633,12 +626,26 @@ func (na *NetworkAllocator) resolveDriver(n *api.Network) (driverapi.Driver, str
 
 	d, _ := na.drvRegistry.Driver(dName)
```

Comment:
We need to ignore all local scope drivers here since a plugin may be local scope and a user might mistakenly ask for it. I know it won't even come here because of the logic in docker/docker but it is better not to assume that and just ignore local scope drivers here.